api/http/api_devauth.go

```diff
@@ -305,7 +322,12 @@ func (d *DevAuthApiHandlers) GetDevicesV2Handler(w rest.ResponseWriter, r *rest.
 		return
 	}
 
-	w.WriteJson(outDevs)
+	_ = w.WriteJson(outDevs)
```

Comment:
How of curiosity, why do you need the `_ =`?

Reply:
I have `golangci-lint` integrated in my editor, and the `errcheck` linter complains that the returned error is not handled.
This syntax indicates that I purposely ignore the error (there's nothing else we really can do in these situations).

Reply:
Thanks for sharing, I'll install the golancgi-lint, too!